apps/ssr-tests/package.json

```diff
@@ -16,15 +16,15 @@
     "@fluentui/react": "^8.0.0-beta.41",
     "@microsoft/load-themed-styles": "^1.10.26",
     "@types/mocha": "^7.0.2",
-    "@types/webpack-env": "1.15.1",
+    "@types/webpack-env": "1.16.0",
     "@fluentui/scripts": "^1.0.0",
     "@fluentui/public-docsite-resources": "1.0.0",
     "just-scripts-utils": "^0.8.3",
     "mocha": "^3.3.0",
-    "raw-loader": "^0.5.1",
+    "raw-loader": "^4.0.2",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "webpack": "4.43.0"
+    "webpack": "^5.12.3"
```

Comment:
Should we lock to a more specific version? (same question on any other deps that might be especially sensitive)

Reply:
not totally sure it matters; we have yarn lock file.

Reply:
The lock file doesn't entirely guarantee that webpack will stay on a particular version; it could be accidentally upgraded at some point along with another change that touches a version of webpack (such as if we go to storybook 6 or start trying yarn-deduplicate again).

Reply:
right, but is that bad? I don't see why that would be bad, or why we would lock that but not everything else that has a caret.

Reply:
I guess from my perspective it's good to lock deps where subtle changes could have significant consequences (typescript being the most obvious example). I've typically put webpack in that category too, but thinking about it more, that may not be correct since our webpack bundles should not actually be what people use in production. So I'm okay with leaving webpack as `^`.